automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/lab/Toolbar.scala

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.dsde.workbench.leonardo.lab
+
+
+trait Toolbar {
+
+  // selects all menus from the header bar
+  lazy val menus: String = "[class='p-MenuBar-item']"
+
+  lazy val menuItems: String ="[class='p-Menu-itemLabel']"
```

Comment:
Is this used anywhere?

Reply:
Same as above. Let me know if you think they should be removed anyways.

Reply:
same as above :)